### PR TITLE
Use Xmlstarlet to modify log4j2.xml

### LINKED ISSUE #3: take care of log4j2 customization during build time
same as duncdrum/exist-docker#5
having exist.log in stdout by default is pretty useful, we should just turn that on for all docker images. 
Right now we still risk having the log4j.xml get out of sync with upstream, by tweaking aureas we don't have to worry about that anymore.

### PR DESCRIPTION
**Pull request by duncdrum**
modify log4j2.xml
close #32 
close #3 

i will edit Readme once #29 is merged

### CODE REVIEW

File: src/conf.xml [hunk 1]
```diff
@@ -198,7 +198,7 @@
         collections, you can also increase the collectionCacheSize value 
     -->
     <db-connection cacheSize="256M" checkMaxCacheSize="true" collectionCache="64M" database="native"
-        files="/exist-data" pageSize="4096" nodesBuffer="1000" cacheShrinkThreshold="10000"
+        files="webapp/WEB-INF/data" pageSize="4096" nodesBuffer="1000" cacheShrinkThreshold="10000"
```
Comment (joewiz) [inline src/conf.xml:201]: Is the `exist-data` volume no longer used?

Comment (duncdrum) [reply src/conf.xml:201]: yes `exist-data` is mapped to `webapp/WEB-INF/data` (the default data-dir).
so from outside the container you still address it as exist-data, inside the container you address it by its internal path.

Comment (joewiz) [reply src/conf.xml:201]: Ah, got it, thanks!

**Comment by duncdrum** [comment]
let me rebase and add the Readme modifications

**Pull request merged**